ivy/functional/backends/paddle/statistical.py

```diff
@@ -328,4 +328,4 @@ def einsum(
     *operands: paddle.Tensor,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    return paddle.einsum(equation, *operands, out=out)
```

Comment:
Hey! Thanks for the great work!
1. `paddle.einsum` doesn't support native `out` argument, please remove `out=out`
2. did you create a todo issue related to the function you're working on? if not, please create one. if yes, please comment your issue number, thanks!